src/nvim/ops.c

```diff
@@ -3644,6 +3644,10 @@ int do_join(size_t count,
    */
   for (t = (linenr_T)count - 1;; t--) {
     cend -= currsize;
+	if(cend == NULL || curr == NULL){
```

Comment:
Yes. I've read it a few times; I'm just new to contributing to open source and trying to figure out how. Please excuse some of the more ignorant questions.

I didn't encounter any conflicts. I wrote my code in a feature branch then merged it into master then created a pull request. I'm just trying to figure out why my build was failing on unit tests, but it seems as though I shouldn't have been running that.